runtime-modules/content-directory/src/lib.rs

```diff
@@ -1867,10 +1904,30 @@ impl<T: Trait> Module<T> {
             DeltaMode::Increment,
         );
 
-        // Add up both net decremental_reference_counter_side_effects and incremental_reference_counter_side_effects
-        // to get one net sideffect per entity.
-        Ok(decremental_reference_counter_side_effects
-            .update(incremental_reference_counter_side_effects))
+        let reference_counter_side_effects = match (
```

Comment:
Can we break this into its own routine, its quite complex. I know it only has one caller right now, but its easier to understand `get_updated_inbound_rcs_delta` if there is an isolated call with a descriptive name for this step.

Reply:
Made an attempt to achieve this here https://github.com/Joystream/joystream/pull/715/commits/26ac2b2ef091c388a222c410e5f3a5575c652811
Though, do not sure about namings in `calculate_updated_inbound_rcs_delta`. Do you have a better idea?

Reply:
I think its fine for now, lets get this wrapped up.